src/items/functions.md

```diff
@@ -160,6 +160,59 @@ attributes], [`must_use`], [the procedural macro attributes], [the testing
 attributes], and [the optimization hint
 attributes].
 
+## Const functions
+
+Functions can be `const`, meaning they can be called from within
```

Comment:
I would somehow explain here that this applies so long as provided arguments (expressions) to the functions explicit parameters (e.g. those within `(...)` but not those within `<...>`) are also valid in const contexts.

Reply:
We don't do that for operators. I don't see how `add(x, y)` differs from `x + y` in the "explain constness"-aspect

Reply:
I would say we should do that for operators in that case. This is the reference, so better be as unambiguous as we can be.